components/widgets/menu/menu_items/menu_item_link.tsx

```diff
@@ -11,7 +11,7 @@ type Props = {
     text: React.ReactNode;
 }
 
-export const MenuItemLinkImpl: React.FC<Props> = ({to, text}: Props) => <Link to={to}>{text}</Link>;
+export const MenuItemLinkImpl: React.FC<Props> = ({to, text}: Props) => <Link to={to}><span className='MenuItem__primary-text'>{text}</span></Link>;
```

Comment:
0/5, but you could just pass the `className` into the `Link` directly to have it put on the `<a>` tag

Reply:
That's cleaner, I'll make a small follow-up PR to improve this @hmhealey